patches/expose_flags.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
+index ff1ed1a98cbfe..7c02d25eef176 100644
+--- a/chrome/browser/about_flags.cc
++++ b/chrome/browser/about_flags.cc
+@@ -4117,6 +4117,10 @@ const FeatureEntry kFeatureEntries[] = {
+ // //tools/flags/generate_unexpire_flags.py.
+ #include "build/chromeos_buildflags.h"
+ #include "chrome/browser/unexpire_flags_gen.inc"
++    {"disable-cross-origin-referrers",
++     flag_descriptions::kDisableCrossOriginReferrersName,
++     flag_descriptions::kDisableCrossOriginReferrersDescription, kOsAll,
++     FEATURE_VALUE_TYPE(switches::kDisableCrossOriginReferrers)},
++    {"cross-origin-trim-referrer",
++     flag_descriptions::kCapReferrerToOriginOnCrossOriginName,
++     flag_descriptions::kCapReferrerToOriginOnCrossOriginDescription, kOsAll,
++     FEATURE_VALUE_TYPE(switches::kCapReferrerToOriginOnCrossOrigin)},
+     {variations::switches::kEnableBenchmarking,
+      flag_descriptions::kEnableBenchmarkingName,
+      flag_descriptions::kEnableBenchmarkingDescription, kOsAll,
+diff --git a/chrome/browser/flag_descriptions.cc b/chrome/browser/flag_descriptions.cc
+--- a/chrome/browser/flag_descriptions.cc
++++ b/chrome/browser/flag_descriptions.cc
+@@ -14,6 +14,11 @@
+ 
+ namespace flag_descriptions {
+
++const char kDisableCrossOriginReferrersName[] =
++    "Clear cross-origin referrers";
++const char kDisableCrossOriginReferrersDescription[] =
++    "Clears referrers when navigating across origins. Disabled by "
++    "default. THIS IS A HARDENED CHROMIUM FEATURE!";
++
++const char kCapReferrerToOriginOnCrossOriginName[] =
++    "Cross-origin referrer trimming";
++const char kCapReferrerToOriginOnCrossOriginDescription[] =
++    "Trims the referrer to just the origin on cross origin navigation. "
++    "Defaults to enabled. THIS IS A HARDENED CHROMIUM EXPOSED FEATURE!";
```

Comment:
Probably can do without the caps :smile: 

just "This is a hardened chromium feature."